hooks/pre/33-pulpcore_assets_permissions.rb

```diff
@@ -0,0 +1,9 @@
+# Prior to Katello 3.18 assets were built by root. Katello 3.18 runs it as pulp
+# and this corrects the permissions.
+unless app_value(:noop)
+  DIRECTORY = '/var/lib/pulp/assets'
+  if File.directory?(DIRECTORY) && File.stat(DIRECTORY).uid == 0
+    require 'fileutils'
+    FileUtils.chown_R('pulp', 'pulp', DIRECTORY)
```

Comment:
This does assume the pulp user exists. I'm wondering if there's a case where that's not true, but `/var/lib/pulp/assets` does exist and owned by root. Backups perhaps? I'll push an update that checks it. Need fix rubocop anyway.